lib/Db.js

```diff
@@ -21,6 +21,35 @@ class Db {
       });
   }
 
+  getProjects() {
+    return this.knex('build')
+      .distinct('projectId')
+      .map(function(row) {
+        return row.projectId;
+      });
+  }
+
+  getBuilds() {
+    return this.knex('build')
+      .distinct('buildId')
+      .map(function(row) {
+        return row.buildId;
+      });
+  }
+
+  getBuildById(buildId) {
+    return this.knex('build')
+      .first()
+      .where({buildId});
+  }
+
+  getBuildSize(buildId) {
+    return this.getBuildById(buildId)
+      .then(function(result) {
+        return result.bytesReal;
+      });
+  }
+
```

Comment:
I think `getProjectSize` would be easy to add here too. We also have to have `getOrganizationSize` but not sure if it's ready to go in here yet.